autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -197,7 +191,7 @@ def __init__(self):
                 ("result", "Hello, World!"),
             ],
             categories={BlockCategory.INPUT, BlockCategory.BASIC},
-            ui_type=BlockUIType.INPUT,
+            block_type=BlockType.INPUT,
```

Comment:
I'm repurposing `BlockUIType` to generic `BlockType`. So we can use it for other means not specific for UI on the backend.